apps/fyle/tasks.py

```
@@ -163,11 +175,66 @@ def create_expense_groups(workspace_id: int, fund_source: List[str], task_log: T
+    :param workspace_id: Workspace id
+    :return: None
+    """
+    skipped_expenses = mark_expenses_as_skipped(Q(), expense_ids, workspace_id)
+    if skipped_expenses:
+        try:
+            post_accounting_export_summary(workspace_id=workspace_id, expense_ids=[expense.id for expense in skipped_expenses])
+        except Exception:
+            logger.exception('Error posting accounting export summary for workspace_id: %s', workspace_id)
+
+
+def group_expenses_and_save(
```

Pylint (3.3.7)

[refactor] Too many branches (13/12)

(R0912)